ghcide/src/Development/IDE/Core/RuleTypes.hs

```diff
@@ -1,48 +1,49 @@
+{-# LANGUAGE DerivingStrategies #-}
 -- Copyright (c) 2019 The DAML Authors. All rights reserved.
 -- SPDX-License-Identifier: Apache-2.0
-
```

Comment:
Please don't include unrelated formatting changes.

Reply:
Thanks for pointing it out. I tried hard not to include them (using Save without Formatting), but as I do it manually, there can be such changes by accident. I'm really longing to see #693 to be resolved...